extension/package.json

```diff
@@ -24,6 +24,7 @@
     "package:with:engine": "bash -c 'vsce package -o ${npm_package_name}-with-engine.vsix --no-update-package-json --no-git-tag-version --yarn ${npm_package_version:0:4}.$(date \"+%Y%m%d%H\")'",
     "publish:next": "bash -c 'vsce publish --yarn --no-update-package-json --no-git-tag-version ${npm_package_version:0:4}.$(date \"+%Y%m%d%H\")'",
     "prepare": "yarn clean && yarn build",
+    "type": "tsc",
```

Comment:
good, but make sure that the noEmit flag is enabled in the tsconfig! otherwise tsc will also transpile the code to js what is not needed for type checking.

Reply:
it's not set and even with noEmit=false I do not get .js output

Reply:
delete the tsconfig.tsbuildinfo ;)